problems/3.conditionals.js

```diff
@@ -19,9 +29,20 @@ function temperatureMessageUsingTernary() {
 * @param {number} age - The age of the user
 * @returns {string} - A message about their age
 */
-function ageMessage() {
+function ageMessage(age) {
+  if(age >= 65){
+    return "You are a senior"}
+    else if(age <= 18){
+      return "You are an adult"}
+      else {
+        return "You are a minor"
+      }
+    
+    
```

Comment:
Looks good. Make sure to use a code formatter. Line 35 should be age >= 18. If they were less than 18 you'd be returning they're and adult.